test/benchmark/schbench/smp1/bench_results/p99_wakeup_latency_smp1.json

```diff
@@ -0,0 +1,17 @@
+{
+    "alert": {
+        "threshold": "130%",
+        "bigger_is_better": false
+    },
+    "result_extraction": {
+        "search_pattern": "\\*",
+        "nth_occurrence": 28,
+        "result_index": 3
+    },
+    "chart": {
+        "title": "[Scheduler] P99 wakeup latency while max-loading the only CPU",
+        "description": "schbench -F 256 -n 5 -r 90",
+        "unit": "\u00b5s",
```

Comment:
What is this character?

@grief8 I am frustrated by json's inability to add comments. Having comments would be particularly useful for "result_extraction", where comments can clarify the meaning of the pattern and give examples of matched results. This "unit" field would also benefit from having a comment.

Reply:
It means microseconds. It can be successfully be translated into μs on the webpage. 

It is possible to add comments but remove them before processing. For example, `"unit": "\u00b5s", # microseconds` and remove them by `sed`.

Reply:
Yes, it's time to adopt a new format. YAML is an excellent choice for its readability and flexibility. We can leverage `yq` to decode and manipulate YAML files, as it is designed specifically for YAML and operates in a manner similar to `jq`, making it a seamless transition for those familiar with JSON processing. It will be discussed in another PR.